cmd/pmm-dump/util.go

```diff
@@ -84,6 +84,91 @@ func getPMMVersion(pmmURL string, c grafana.Client) (string, string, error) {
 	return resp.Server.Version, resp.Server.FullVersion, nil
 }
 
+func getPMMServices(pmmURL string, c grafana.Client) ([]dump.PMMServerService, error) {
+	type servicesResp map[string][]struct {
+		ID     string `json:"service_id"`
+		Name   string `json:"service_name"`
+		NodeID string `json:"node_id"`
+	}
+	type nodeResp struct {
+		Generic struct {
+			Name string `json:"node_name"`
+		} `json:"generic"`
+	}
+	type agentsResp map[string][]map[string]interface{}
+
+	// Services
+
+	statusCode, body, err := c.Post(fmt.Sprintf("%s/v1/inventory/Services/List", pmmURL))
+	if err != nil {
+		return nil, err
+	}
+	if statusCode != fasthttp.StatusOK {
+		return nil, fmt.Errorf("non-ok status: %d", statusCode)
+	}
+	serviceResp := new(servicesResp)
+	if err = json.Unmarshal(body, serviceResp); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal response: %s", err)
+	}
+
+	services := make([]dump.PMMServerService, 0)
+	for _, v := range *serviceResp {
+		for _, serviceV := range v {
+			service := dump.PMMServerService{
+				Name:   serviceV.Name,
+				NodeID: serviceV.NodeID,
+			}
```

Comment:
```suggestion
	for _, serviceType := range *serviceResp {
		for _, service := range serviceType {
			newService := dump.PMMServerService{
				Name:   service.Name,
				NodeID: service.NodeID,
			}
```
Please make it more readable